Sources/TuistKit/Services/RunService.swift

```diff
@@ -0,0 +1,142 @@
+import Foundation
+import RxSwift
+import TSCBasic
+import TuistAutomation
+import TuistCore
+import TuistSupport
+
+enum RunServiceError: FatalError {
+    case appNotFound(targetName: String)
+    case simulatorNotFound
+    case targetNotFound(forSchemeName: String)
+
+    var description: String {
+        switch self {
+        case let .appNotFound(targetName):
+            return "Couldn't find any apps built for \(targetName)"
+        case .simulatorNotFound:
+            return "Could't find simulator"
+        case let .targetNotFound(forSchemeName):
+            return "Couldn't find \(forSchemeName)"
```

Comment:
I'd extend it to include the list of schemes. That makes it easier for the user and for us to debug the issue:

```
Couldn't find scheme '\(schemeName)' in the project. The list of buildable schemes are: ....`
```